Jot/TrackingConfiguration.cs

```diff
@@ -35,15 +35,22 @@ public sealed class TrackingConfiguration
 
         /// <summary>
         /// The identity of the target. This severs to identify which stored data belongs to which object. If not specified,
-        /// only the type name is used, which is fine for singletons.
+        /// only the type name is used, which is fine for singletons. (check out NamingScheme)
         /// </summary>
         public string Key { get; set; }
 
         /// <summary>
-        /// The identity of the target. This severs to identify which stored data belongs to which object. If not specified,
-        /// check out the Key definition. If specified the Key is used to indetification. In case Key is not defined, this parameter is ignored.
+        /// The identity of the target. This severs to identify which stored data belongs to which object. 
```

Comment:
Not exactly just didn´t format comments yet, because don´t know if this enum will stay in this class or will be moved...